docs/src/docker.md

```diff
@@ -165,3 +164,18 @@ The image will be tagged as `playwright:localbuild-jammy` and could be run as:
 docker run --rm -it playwright:localbuild /bin/bash
 ```
 
+## Development
+* langs: python
+
+### Build the image
+
+To run Playwright inside Docker, you need to have Python, [Playwright browsers](./browsers.md#install-browsers) and [browser system dependencies](./browsers.md#install-system-dependencies) installed. See the following Dockerfile:
+
+```Dockerfile
+FROM python:3.12-bookworm
+
+RUN pip install playwright==@%%VERSION%% && \
+    playwright install --with-deps
+```
+
+Note: official images published to [Microsoft Artifact Registry] are built using [`//utils/docker/build.sh`](https://github.com/microsoft/playwright-python/blob/main/utils/docker/build.sh) script.
```

Comment:
This is our implementation details, we shouldn't be advertising it. Those who curious can figure it out by looking at the sources already but I would not recommended to general clients.

Reply:
Happy to remove this in JS + Python.

Reply:
Let's remove.